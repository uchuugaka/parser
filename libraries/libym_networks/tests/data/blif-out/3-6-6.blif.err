"3-6-6.blif" line: 7, column: 15 [ER_MLTDEF01]: x: Defined more than once. Previsous Definition is "3-6-6.blif" line: 6, column: 10.
Error in reading 3-6-6.blif
