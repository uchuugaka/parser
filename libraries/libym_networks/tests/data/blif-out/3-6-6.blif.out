.model test1
	["3-6-6.blif" line: 3, columns:1-6]
	["3-6-6.blif" line: 3, columns:8-12]
  inputs elem: a
	["3-6-6.blif" line: 4, column: 9]
  inputs elem: b
	["3-6-6.blif" line: 4, column: 11]
  outputs elem: x
	["3-6-6.blif" line: 5, column: 10]
.latch a x -1
	["3-6-6.blif" line: 6, column: 8]
	["3-6-6.blif" line: 6, column: 10]
	["3-6-6.blif" line: 6, column: 11]
.gate inv4x begin
	["3-6-6.blif" line: 7, columns:1-5]
	["3-6-6.blif" line: 7, columns:7-11]
error_exit
